tests/std/tests/VSO_0000000_regex_use/test.cpp

```diff
@@ -546,6 +546,17 @@ void test_VSO_226914_word_boundaries() {
     aWordAny.should_search_fail("aa", match_not_bow | match_not_eow);
 }
 
+void test_gh_731() {

```

Comment:
This should be followed by a comment mentioning the bug's title (like `test_gh_993()` immediately below).